### PR TITLE
Onboard Decision Component to Axon-ivy market

### PR DESCRIPTION
**Pull request by nntthuy-axonivy**

### CODE REVIEW

**Comment by nntthuy-axonivy** [comment]
Hi @nvvi @ptanhaxon,
Thanks for your amazing contribution!
As discussed before, I forked the code from your repo https://github.com/axonivy-professional-services/decision-component-utils to my market repo and created this pull request for onboarding new connector.

I will start reviewing the product now, if I find any concerns or questions, I will mention you in the comment section. Thanks for your understanding!

FYI @ivy-sgi

**Review by nntthuy-axonivy — COMMENTED** [review]
Hi @nvvi @ptanhaxon, could you check my concerns and handle them? After importing the project successfully, I'll revise codes once again. Thank you!

File: decision-component-utils-demo/src/com/axonivy/utils/decisioncomponent/demo/dao/TicketRequestDAO.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.hibernate.Hibernate;
+
+import com.axonivy.utils.decisioncomponent.demo.entities.TicketRequest;
+import com.axonivy.utils.decisioncomponent.demo.entities.TicketRequest_;
+import com.axonivy.utils.persistence.dao.AuditableDAO;
+import com.axonivy.utils.persistence.dao.CriteriaQueryContext;
+
+public class TicketRequestDAO extends AuditableDAO<TicketRequest_, TicketRequest> implements BaseDAO {
+
+	private static final TicketRequestDAO INSTANCE = new TicketRequestDAO();
```
[the remark this answers is not in the document]
Comment (nvvi) [reply decision-component-utils-demo/src/com/axonivy/utils/decisioncomponent/demo/dao/TicketRequestDAO.java]: this is a java constant, so it should be uppercase..

File: decision-component-utils/webContent/layouts/includes/exception.xhtml
[the remark this answers is not in the document]
Comment (nvvi) [reply decision-component-utils/webContent/layouts/includes/exception.xhtml:1]: this is default from ivy, so i will not re-format this file.

File: decision-component-utils/webContent/layouts/includes/exception-details.xhtml
[the remark this answers is not in the document]
Comment (nvvi) [reply decision-component-utils/webContent/layouts/includes/exception-details.xhtml:1]: this file auto generated from designer, so i think we don't need to format it

File: decision-component-utils/webContent/layouts/basic-10.xhtml
[the remark this answers is not in the document]
Comment (nvvi) [reply decision-component-utils/webContent/layouts/basic-10.xhtml:1]: this file auto generated from designer, so i think we don't need to format it

**Pull request merged**